config/helm/deployment-labels.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Patch for setting `app.kubernetes.io/component` that we need for `management-cluster-admission`.